### PR TITLE
MC-11448: adding documentation for create and update idp

### PR DESCRIPTION
**Pull request by evanluc**
### Fixes [MC-11448](https://cloud-ops.atlassian.net/browse/MC-11448)

Added docs for create and update of identity providers.

### CODE REVIEW

File: source/includes/administration/_authentication.md [hunk 1]
```diff
@@ -59,8 +59,176 @@ Attributes | &nbsp;
...
+`id`<br/>*UUID* | The id of the identity provider.
+`provider`<br/>*string* | The name of the provider. Possible values include the default providers (e.g GOOGLE), or CUSTOM for a custom user-defined provider.
+`displayName`<br/>*string* | The display name of the identity provider that will appear on the login screen.
+`connectionName`<br/>*string* | The connection name of the identity provider.
+`type`<br/>*string* | The type of authentication protocol. Possible values: OIDC, SAML.
+`parameters`<br/>*Array* | A list of parameters associated with the identity provider.
+
+Optional | &nbsp;
+---------- | -----------
+`logo`<br/>*string* | A base64 encoded data url or url to an image for the logo to display on the login screen.
```
Comment (franzpgarcia) [inline source/includes/administration/_authentication.md]: isn't it only optional if you have a non default provider? also, you should mention that default providers bring they're own values/

File: source/includes/administration/_authentication.md [hunk 2]
```diff
@@ -59,8 +59,176 @@ Attributes | &nbsp;
...
 `type`<br/>*string* | The type of authentication protocol. Possible values: OIDC, SAML.
 `parameters`<br/>*Array* | A list of parameters associated with the identity provider.
+
+<!-------------------- CREATE IDPS -------------------->
+
+#### Create identity provider
+
+`POST /identity_providers`
+
+Create a new identity provider
```
Comment (lamminade) [inline source/includes/administration/_authentication.md]:
```suggestion
Create a new identity provider.
```
just a missing period

**Review by lamminade — APPROVED** [review]

File: source/includes/administration/_authentication.md (hunk 1 above)
Comment (evanluc) [reply source/includes/administration/_authentication.md]: https://github.com/cloudops/cloudmc-api-docs/pull/153/commits/91a783383ac46482d2bc8ba762081cb266376e64

Comment (evanluc) [reply source/includes/administration/_authentication.md]: https://github.com/cloudops/cloudmc-api-docs/pull/153/commits/f1981cc0deadb19e62d0c45174765cb0e98b884c

**Review by franzpgarcia — APPROVED** [review]

**Pull request merged**